### ISSUE TITLE
Updated for React version 15.5

### ISSUE THREAD

**Issue by pionl** [kentcdodds/react-test-context-provider#1]
Hi, 

could you update the package for newer react?

```
console.error node_modules/fbjs/lib/warning.js:36
    Warning: Accessing PropTypes via the main React package is deprecated. Use the prop-types package from npm instead.

  console.error node_modules/fbjs/lib/warning.js:36
    Warning: ContextProvider: React.createClass is deprecated and will be removed in version 16. Use plain JavaScript classes instead. If you're not yet ready to migrate, create-react-class is available on npm as a drop-in replacement.

```

**Comment by kentcdodds** [comment]
Hi @pionl,
I didn't realize folks were using this! I found out later that there are many other tools to do this. If you use enzyme, this is provided ([for example](https://github.com/airbnb/enzyme/blob/master/docs/api/mount.md#mountnode-options--reactwrapper)).

If you'd like to keep using this, would you like to take ownership? I can give you push access to the repo and publish access to the npm package.

**Comment by sibelius** [comment]
I think the idea of this package is good

We should refactor to use React.Component instead of createClass

**Comment by kentcdodds** [comment]
That's fine with me. Do you want the package? I'll give it to you 😀

**Comment by pionl** [comment]
Hi @kentcdodds, thank you for the response. At this moment I don't have to much time, but next week I could try to find the time. If @sibelius prefers to make the changes, I'm happy to pass it :)

Have a great day,

**Comment by alanbsmith** [comment]
Oh, I should've read this before submitting a PR. whoops.

**Comment by alanbsmith** [comment]
Closing. This issue was resolved with #2 .
